tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1344,10 +1344,14 @@ public:
 
       RResultPtr<RInterface<RLoopManager>> resPtr;
 
-      auto retrieveTypeID = [](const std::string &colName, const std::string &colTypeName) -> const std::type_info * {
+      auto retrieveTypeID = [](const std::string &colName, const std::string &colTypeName,
```

Comment:
Unless this needs to be extended to other backends, a string is a bit much to switch between tree and RNT, wouldn't you say? For these two alternatives, a bool would do.